pages/add/add.jsx

```diff
@@ -1,12 +1,161 @@
+                                         header={`Please sign in to add a post`}
+                                         externalLink={user.getLoginURL(redirectUrl)}
+                                         linkText={`Sign in`}
+                                         onClick={this.handleSignInBtnClick}>
+                            <p>Please note, only Mozilla staff can login now as we test this new platform.</p>
+                          </HintMessage>);
+
+
+    let failurePrompt= ( <HintMessage imgSrc={`/assets/svg/icon-user.svg`}
+                                      header={`Sign in failed`}
+                                      internalLink={`/featured`}
+                                      linkText={`Explore featured`}>
+                          <p>Only Mozilla staff can login now as we test this new platform. Check back soon!</p>
+                        </HintMessage>);
+
+    let contentForLoggedInUser = (
+      <div>
+        <h1>Share with the Network</h1>
+        <p>Do you have something to share? If it might be useful to someone in our network, share it here! Pulse includes links to products and software tools, research reports and findings, think pieces, white papers, interviews, and curricula. If it might be useful, share it … at any stage or fidelity.</p>
+        <form action="" method="post">
+          <h2>Basic Info</h2>
+          <div className="posted-by">
+            <p>Posted by:&nbsp;<span className="user-full-name">{user.username}</span></p>
+            <p className="log-out-prompt">Not you? <button className="btn btn-link" onClick={this.handleLogOutBtnClick}>Sign out</button>.</p>
+          </div>
+          <fieldset hidden>
+            <label className="form-control-label">
+              Posted by:&nbsp;
+              <input name="" type="text" className="form-control" value="" readOnly />
+            </label>
+          </fieldset>
+          <label className="form-control-label">
+            Title of the project<em className="required">(required)</em>
+            <input name="title" type="text" placeholder="Title" className="form-control" />
+          </label>
+          <label className="form-control-label">
+            URL<em className="required">(required)</em>
+            <input name="" type="url" placeholder="https://example.com" className="form-control" />
+          </label>
+          <label className="form-control-label">
+            Describe what you are sharing. Keep it simple and use plain language.
+            <textarea name="description" placeholder="Description" className="form-control" />
+          </label>
+          <h2>Optional Details</h2>
+          <fieldset className="creators">
+            <legend>Who are the creators? This could be staff, contributors, partners…</legend>
+            { creatorFields }
+            <button onClick={this.handleCreatorClick} className="btn btn-link">&#43; Add another creator</button>
```

Comment:
note that the creatorFields still add "new fields" inline instead of as blocks:
![screenshot 152](https://cloud.githubusercontent.com/assets/177243/22392041/3df8b54c-e4aa-11e6-89eb-6375b7edb9a4.png)